servant-foreign/src/Servant/Foreign/Internal.hs

```diff
@@ -7,7 +7,7 @@
 -- arbitrary programming languages.
 module Servant.Foreign.Internal where
 
-import           Control.Lens hiding (cons, List)
+import           Control.Lens (makePrisms, makeLenses, Getter, (&), (<>~), (.~), (%~))
```

Comment:
I agree about lens in particular, but we were getting a warning since `List` didn't exist, so it was this or CPP